libraries/AP_HAL_ChibiOS/hwdef/CubeRedSecondary-IO/hwdef.dat

```diff
@@ -0,0 +1,170 @@
+# hw definition file for processing by chibios_hwdef.py
+# for H757
+
+# MCU class and specific type
+MCU STM32H7xx STM32H757xx
+
+define CORE_CM7
+define SMPS_PWR
+
+env OPTIMIZE -O3
+
+# crystal frequency
+OSCILLATOR_HZ 24000000
+
+# board ID for firmware load
+APJ_BOARD_ID 1070
+
+FLASH_SIZE_KB 2048
+
+# bootloader is installed at zero offset
+FLASH_RESERVE_START_KB 128
+
+# the location where the bootloader will put the firmware
+# the H743 has 128k sectors
+FLASH_BOOTLOADER_LOAD_KB 128
+
+define HAL_LED_ON 0
+
+define IOMCU_FW TRUE
+define AP_FASTBOOT_ENABLED 1
+IOMCU_FW 1
+
+define HAL_USE_RTC FALSE
+define HAL_NO_UARTDRIVER TRUE
+define HAL_LOGGING_ENABLED 0
```

Comment:
Why are these in here?  These are the default for io firmware... (defaults_periph.h)

Reply:
They are not actually, only HAL_LOGGING_ENABLED is. In anycase, I copied this from iomcu/hwdef.inc

Reply:
> They are not actually, only HAL_LOGGING_ENABLED is. In anycase, I copied this from iomcu/hwdef.inc

`HAL_NO_UARTDRIVER` no longer exists.

If `HAL_USE_RTC` needs a default to "off" for IOMCU (and it does!) that default should go into `defaults_iofirmware.h`.  The HAL_LOGGING_ENABLED in `iomcu/hwdef.inc` is redundant as it already appears in `defaults_iofirmware.h`